### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Fix linter
-  #74
- #130 
- #127

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: manifest.json [hunk 1]
```diff
@@ -3,19 +3,23 @@
...
-        "fr": "Sauvegardez votre serveur sur un serveur distant avec Borg."
+        "en": "Backup your server on a host server using Borg",
+        "fr": "Sauvegardez votre serveur sur un serveur distant avec Borg"
     },
-    "version": "1.1.16~ynh28",
+    "version": "1.1.16~ynh29",
     "url": "https://borgbackup.readthedocs.io",
+    "upstream": {
+        "license": "BSD-3-Clause",
+        "website": "https://borgbackup.readthedocs.io",
```
Comment (fflorent) [inline manifest.json:13]:
```suggestion
        "website": "https://borgbackup.readthedocs.io"
```

**Comment by zamentur** [comment]
!testme

**Comment by alexAubin** [comment]
!testme

**Comment by ericgaspar** [comment]
maybe we should merge this @zamentur

**Review by alexAubin — APPROVED** [review]

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**